src/views/dashboard/Dashboard.js

```diff
@@ -2,37 +2,37 @@ import React, { useState } from 'react'
 import styled from 'styled-components'
 import { useHistory } from 'react-router-dom'
 import { ViewContainer, ActionsContainer, Fab } from '../../components'
-import { ReactComponent as Calendar } from '../../components/icons/calendar.svg';
-import { ReactComponent as Team } from '../../components/icons/team.svg';
-import { ReactComponent as Services } from '../../components/icons/cardiovascular.svg';
+import { ReactComponent as Calendar } from '../../components/icons/calendar.svg'
+import { ReactComponent as Team } from '../../components/icons/team.svg'
+import { ReactComponent as Services } from '../../components/icons/cardiovascular.svg'
 import { ROUTES } from '../../constants'
 
 const DashboardCardsContent = (props) => {
   return <div>{props.children}</div>
 }
 
 const PlannersContainer = styled.div`
-  background-color: #084B83;
+  background-color: #084b83;
   width: 100%;
```

Comment:
Styling guide with colours of the app.

Reply:
TBD

Reply:
An Issue was created and be implemented soon:
https://github.com/Phlammariont/happiny/issues/18